### ISSUE TITLE
JavaScript Error when trying to open the ban modal

### ISSUE THREAD

**Issue by wonderbeel** [FriendsOfFlarum/ban-ips#11]
When I try to open the ban modal from a user page nothing happens and in the console there is this error
```
Uncaught ReferenceError: post is not defined
    onclick addBanIPControl.js:38
    handleEvent render.js:888
```

It seems to be caused by this line https://github.com/FriendsOfFlarum/ban-ips/blob/master/js/src/forum/addBanIPControl.js#L38 I don't know mithril too much but at least to me it seems that `post` isn't available in the function scope.

**Comment by dsevillamartin** [comment]
Huh, yes that is definitely a bug. Thanks for catching it!

**Comment by dsevillamartin** [comment]
Fixed in v1.0.1
